run.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 set -eo pipefail
 
-# First remove everything from the folder
-find . -mindepth 1 -maxdepth 1 ! -name 'run.sh' -type f  -exec rm -f {} +
-find . -mindepth 1 ! -name 'jobs' -type d -exec rm -rf {} +
+# First remove everything from the workdir
+rm -rf /data/*
```

Comment:
This is not a good move. If the user attaches an already existing pv/pvc we will wipe it.
Can we please remove it?

Reply:
Removed

Reply:
it is still there 🙀